src/app-layout/__tests__/split-panel.test.tsx

```diff
@@ -109,11 +109,15 @@ for (const theme of ['refresh', 'classic']) {
             onSplitPanelPreferencesChange={noop}
           />
         );
-        expect(wrapper.findSplitPanel()!.findOpenButton()).not.toBeNull();
-        act(() => wrapper.findSplitPanel()!.findOpenButton()!.click());
-        expect(wrapper.findSplitPanel()!.findOpenButton()).toBeNull();
-        act(() => wrapper.findSplitPanel()!.findCloseButton()!.click());
-        expect(wrapper.findSplitPanel()!.findOpenButton()).not.toBeNull();
+        expect(wrapper.findSplitPanelOpenButton()).not.toBeNull();
+        wrapper.findSplitPanelOpenButton()!.click();
+        if (position === 'side' && theme === 'refresh') {
```

Comment:
This test was a false positive. It was supposed to test bottom and side split panel position, but because JSDOM defaults it always assumed "forced bottom" position.

I fixed this long ago for classic in #1952 and now I reused some of that code for refresh, which made this test work correctly.

The expected design requirement is that in refresh and side position, the open button _stays visible_. So the test matches the original requirements.